sql/nm_spawn_points.sql

```diff
@@ -252,6 +252,8 @@ INSERT INTO `nm_spawn_points` VALUES (16806227,49,-549.802,-8.944,-24.848);
 INSERT INTO `nm_spawn_points` VALUES (16806249,0,-490.195,-4.503,145.451); -- Ambusher Antlion
 INSERT INTO `nm_spawn_points` VALUES (16822423,0,4.609,11.472,-101.235); -- Bugbear Strongman (west)
 INSERT INTO `nm_spawn_points` VALUES (16822427,0,60.728,15.487,-118.886); -- Bugbear Strongman (east)
+INSERT INTO `nm_spawn_points` VALUES (16851267,0,-44.3170,0,-242.987); -- Satiator
+-- INSERT INTO `nm_spawn_points` VALUES (16851267,1,-205.880,-0.500,268.850); -- Satiator on inaccessable island. 
```

Comment:
This wasn't the part that needs to be put in commented out. As it stands now, neither sql row in this table is at all being used at all. Add the function calls, and leave them in commented out, then this PR is ready to pass.